Drivers/Src/StatusLeds.c

```diff
@@ -0,0 +1,99 @@
+#include "StatusLeds.h"
+
+/**
+ * @brief Initializes Status LED pins
+ * 
+ */
+void Status_Leds_Init(void){
+    BSP_GPIO_Init(OS_FAULT_PORT, OS_FAULT, OUTPUT, false);
+    BSP_GPIO_Init(MOTOR_CONTACTOR_PORT, MOTOR_CONTACTOR, OUTPUT, false);
+    BSP_GPIO_Init(MOTOR_PRCHG_BYPASS_PORT, MOTOR_PRCHG_BYPASS, OUTPUT, false);
+    BSP_GPIO_Init(ARRAY_PRCHG_BYPASS_PORT, ARRAY_PRCHG_BYPASS, OUTPUT, false);
+    BSP_GPIO_Init(MOTOR_CTRL_FAULT_PORT, MOTOR_CTRL_FAULT, OUTPUT, false);
+    BSP_GPIO_Init(BPS_FAULT_PORT, BPS_FAULT, OUTPUT, false);
+    BSP_GPIO_Init(CONTROLS_FAULT_PORT, CONTROLS_FAULT, OUTPUT, false);
+    BSP_GPIO_Init(CRUISE_IND_PORT, CRUISE_IND, OUTPUT, false);
+} 
+
+/**
+ * @brief   Writes data to a specified pin
+ * @param   led The led to write too
```

Comment:
```suggestion
 * @param   led The led to write to
```
Might as well make the spelling as great as the code is :)